Docs: Adds initial status history docs (#35243)
grafana/grafana: docs/sources/panels/visualizations/status-history.md

+++
title = "Status history"
description = "Status history visualization"
keywords = ["grafana", "docs", "status history", "panel"]
weight = 900
+++

# Status history

> This panel shows periodic state over time. Each field or series is rendered as a horizontal row. Boxes are rendered and centered around each value. 

Reviewer: ```suggestion
The Status history visualization shows periodic states over time. Each field or series is rendered as a horizontal row. Boxes are rendered and centered around each value. 
```

{{< figure src="/static/img/docs/status-history-panel/status-history-example-v8-0.png" max-width="1025px" caption="Status history example" >}}

## Supported data 

> This panel works with string, boolean and numerical fields or time series. A time field is required. You can use value mappings to color strings or assign text values to numerical ranges.  

Reviewer: ```suggestion
Status history visualization works with string, boolean and numerical fields or time series. A time field is required. You can use value mappings to color strings or assign text values to numerical ranges.  
```
